### ISSUE TITLE
No spoken dialog in Might and Magic - World of Xeen in v0.78.1 (was working in v0.77.1)

### ISSUE THREAD

**Issue by codeflorist** [dosbox-staging/dosbox-staging#1937]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### What Operating System are you using?

Windows 10

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Using v0.78.1 the spoken dialogue in World of Xeen CD version is not working. Strangely enough, not even the lips of the characters are moving. It is working in v0.77.1.

Tested with both the GOG version as well as the original CD version.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Start a new game
2. Enter the tent of Gunther the Mayor, immediately after the starting position
3. Gunther is not talking and his lips are not moving

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by kcgen** [comment]
Tested with 0.78 and the current `main` build: 

[mm6_cd1_cdda.webm](https://user-images.githubusercontent.com/1557255/190883265-ff365a6d-c193-410e-8bab-223121107da7.webm)

Performed a new installation from BIN/CUE image, using all default settings except for setting cycles to ~16000, to give it smooth mouse motion.

```ini
[cpu]
cycles = 16000
```

**Comment by codeflorist** [comment]
@kcgen thanks for cross-checking! i dug a little deeper and i think i've found the culprit. the problem only occurs, if you mount both CDs (to be able to switch them with CTRL-F4).  it works, if you only mount one image.

here is my doxbox.conf:
```ini
[cpu]
cycles=16000

[autoexec]
mount C "HDD"
imgmount d "ISO\xeen_CD1.cue" "ISO\xeen_CD2.cue" -t iso
```

i've launched it with the debug-exe, and when klicking che mayor's tent, it spams the following messages:
```
339977812: MISC:MSCDEX: INT 2F 1510 BX= FB12 CX=0003
339977812: MISC:MSCDEX: Driver Function 85
339977812: MISC:MSCDEX: Status : 0100
```

with v0.77.1 the problem does not occur.

**Comment by kcgen** [comment]
Nice digging, @codeflorist!
This bodes well for bisecting where this issue was introduced. 
I've tagged it for 0.80, but will plan to backport into 0.79.1 if it's a small fix (as we're focusing on getting 0.79 released right now).

**Comment by kcgen** [comment]
@codeflorist , pushed a fix, when you can try the latest `main` build.

**Comment by codeflorist** [comment]
@kcgen it fixed in `v0.80.0-alpha-10-g383c9`. many thanks!

**Comment by kcgen** [comment]
Thanks for the quick check, @codeflorist.
Please feel free to stick around and help with more, if you'd like!
